components/common-css/container.css

```diff
@@ -0,0 +1,74 @@
+@import url(../../docs-css/reset.css);
+.component{
+    display: flex;
+    height: 90vh;
+    
+}
+.preview-container{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 69% ;
+    padding: 1rem;
+    border-radius: 10px;
+    background-color: var(--card-bg);
+    margin:1rem;
+    box-shadow: var(--box-shadow);
+}
+.doc-container{
+    padding: 3rem;
+    width: 100%;
+    overflow-y:auto;
+}
+
+
+.description-area{
+    display: flex;
+    flex-direction: column;
+    font-family: 'Rubik', sans-serif;
+    color: var(--primary);
+    width: 100%;
+}
+.description-heading{
+  font-size: 1.5rem;
+  font-weight: 500;
+  margin: 1rem;
+}
+.description{
+ font-size: 1.1rem;
+ color: var(--font-primary);
+ margin: 1rem;
+ margin-top: 0;
+ margin-bottom: 0.2rem;
+}
+.code pre{
+    height: 100%;
+    width: 100%;
+    margin:0;
+}
+code{
+margin: 1rem;
+margin-top: 0;
+margin-bottom: 0;
+background-color: var(--primary);
+font-size: 0.8rem;
+letter-spacing: 1px;
+line-height: 1.2rem;
+border-radius: 7px;
+padding-left: 1rem !important; 
+width: 70%;
+}
+
+.description-list{
+    font-size:1rem;
+}
+.inverted{
+    color: #1D9CEB;
```

Comment:
Please try using either variables everywhere or color hex/ names for consistency.